### ISSUE TITLE
FF Windows App hangs when selecting variables from properties panel popup (old issue is back)

### ISSUE THREAD

**Issue by efenech** [FlutterFlow/flutterflow-issues#3107]
### Can we access your project?

- [X] I give permission for members of the FlutterFlow team to access and test my project for the sole purpose of investigating this issue.

### Current Behavior

This seems to be an old issue that previous vesions of the FF Windows App had, that was resolved and is now back with v4.1.59. When selecting any of the variables from the properties panel (e.g. selecting an IF condition) the app hangs for about 30-60 seconds. Obviously I can't show you the effect, but the image shows you an example of where it occurs. 
<img width="416" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/88217100/0e3502b3-0a2d-411f-9141-0075709c3b88">

Cheers

### Expected Behavior

Variable selection should be instant (or near instant)

### Steps to Reproduce

1. Click on any of the dynamic options on the properties panel, e.g. Text
2. Select a Conditional Value
3. Select First Value
4. App hangs for about 30-60 seconds.
5. Option becomes available for selection.

### Reproducible from Blank

- [ ] The steps to reproduce above start from a blank project.

### Bug Report Code (Required)

IT4gk+bI0IhgrsBE7r6JcO9V+SoWH2InbIc0sd4bGA0dCJzuPLMQdvWlbktDQsDiY1ZUMESmmkYB+aCOkfzDOuwoNTiVUb1x1s9icBXLfEKua5OBPKmadkB6L+VVCFS90KWzgx1TKu5qcmwd12WpONnKfD/ZO6bbImdISq/LZO4=

### Visual documentation

<img width="416" alt="image" src="https://github.com/FlutterFlow/flutterflow-issues/assets/88217100/0e3502b3-0a2d-411f-9141-0075709c3b88">

### Environment

```markdown
- FlutterFlow version: 4.1.59
- Platform: Windows Desktop App
- Browser name and version: Version 125.0.6422.142 (Official Build) (64-bit)
- Operating system and version affected: Windows 11 Pro 13th Gen Intel(R) Core(TM) i9-13900HX   2.20 GHz 64.0 GB (63.7 GB usable) 64-bit operating system, x64-based processor
```

### Additional Information

Slows down development.

**Comment by rzambroni** [comment]
Hey @efenech, thanks for the report! i'm not currently being able to replicate the issue you are describing. At least on my windows laptop and with a blank project it is working normally. 

Is this happening consistently and also in a blank project? or is it specific to the project you are currently working on?

Thanks.

**Comment by efenech** [comment]
> Hey @efenech, thanks for the report! i'm not currently being able to replicate the issue you are describing. At least on my windows laptop and with a blank project it is working normally.
> 
> Is this happening consistently and also in a blank project? or is it specific to the project you are currently working on?
> 
> Thanks.

I haven't tried it in a blank project, just the one I'm working on. As for it happening consistently, it was, but I rebooted after installing the latest version and experiencing the issue, and it now seems to be more random and less frequent. If it helps, I can note down each time it occurs and where and what I was doing.

**Comment by rzambroni** [comment]
If you can quickly make a blank project and test if it hangs there as well it would be ideal, we can discard weather it is project related or platform related and then see from there what might be the cause in your project (if that is the case)

**Comment by efenech** [comment]
> If you can quickly make a blank project and test if it hangs there as well it would be ideal, we can discard weather it is project related or platform related and then see from there what might be the cause in your project (if that is the case)

Hi, I created a new blank project as you suggested and tested to see if I could repeat the issue. I tried a number of different things, and at this time, it seems to be working without any problem. I've also noticed the issue hasn't occured in my actual project since my last comment to you. I'm a bit stumped as to why it was happening, and I don't want to waste your time on this any further, so I'll chalk it up as a local environment issue - whatever it was. Cheers for you prompt response.
